templates/repo/settings/options.tmpl

```diff
@@ -80,9 +80,26 @@
 					</div>
 					<div class="field {{if .Err_MirrorAddress}}error{{end}}">
 						<label for="mirror_address">{{.i18n.Tr "repo.mirror_address"}}</label>
-						<input id="mirror_address" name="mirror_address" value="{{.Mirror.FullAddress}}" required>
+						<input id="mirror_address" name="mirror_address" value="{{.Mirror.AddressNoCredentials}}" required>
 						<p class="help">{{.i18n.Tr "repo.mirror_address_desc"}}</p>
 					</div>
+					<div class="ui accordion optional field">
+						<label class="ui title {{if .Err_Auth}}text red active{{end}}">
+							<i class="icon dropdown"></i>
```

Comment:
Would remove that `<i>` as the section is not expandable

Reply:
Or actually find out why semantic-ui does not make it interactive, something in the HTML I assume.

Reply:
I think the issue is that the HTML is unbalanced. The `<label for="">` tag below should probably be closed on the same line.

Reply:
Heya! I have no idea how that unbalanced label sneaked in.

So the idea of sticking a label in there was to keep the formatting of collapsable section title the same - it looked weird to my eyes when it was different to all the other bits. I'll have to see if closing that label has changed that back.